src/main/java/org/example/domain/model/MedicalVisit.java

```diff
@@ -0,0 +1,54 @@
+package org.example.domain.model;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+public class MedicalVisit {
+
+    private int id;
+    private LocalDate date;
+    private LocalTime time;
+    private final Doctor doctor;
+    private Patient patient;
+
+    public MedicalVisit(LocalDate date, LocalTime time, Doctor doctor, Patient patient) {
+        this.id = id;
+        this.date = date;
+        this.time = time;
+        this.doctor = doctor;
+        this.patient = patient;
+    }
+
+    public LocalDate getDate() {
+        return date;
+    }
+
+    public LocalTime getTime() {
+        return time;
+    }
+
+    public Doctor getDoctor() {
+        return doctor;
+    }
+
```

Comment:
hkjhk

Reply:
to tutaj miało być :)
Lepiej rozbić to na datę i godzinę osobno
coś w stylu
ex. constructor
public MedicalVisit(int idx, int doctorId, int patientId, int month, int day, int hour, int minute, VisitStatus status)
zwyczajnie łatwiej na tym pracować